spec/requests/api/v1/listings/listing_limitation_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+RSpec.describe 'POST /api/v1/listings', type: :request do
+  let!(:landlord) { create(:user, role: 'registered') }
+  let!(:listing) { 5.times { create(:listing, landlord: landlord) } }
```

Comment:
Why create 5 listings for the registered user?

Reply:
Good point